rust/datafusion/src/execution/physical_plan/math_expressions.rs

```diff
@@ -92,7 +122,7 @@ mod tests {
             .build()?;
         let ctx = ExecutionContext::new();
         let plan = ctx.optimize(&plan)?;
-        let expected = "Projection: sqrt(CAST(#c0 AS Float64))\
+        let expected = "Projection: sqrt(CAST(#c0 AS Float32))\
```

Comment:
notice here how i8 is now mapped to float32, not float64 as we give preference to smaller types.